legal-api/src/legal_api/resources/v2/business/business_parties.py

```diff
@@ -43,8 +43,13 @@ def get_parties(identifier, party_id=None):
             HTTPStatus.UNAUTHORIZED,
         )
 
+    business_id = business.id
+    if business.is_owned_by_legal_entity_person or business.is_owned_by_legal_entity_org:
+        business_id = business.legal_entity_id
+    elif business.is_owned_by_colin_entity:
+        business_id = business.colin_entity_id
```

Comment:
so this won't work.

There are no `entity_role` entries for all the SP scenarios.

The code needs to be updated such that if `business.entity_type` == `SP`, party json is still returned.

Right now if you run this endpoint locally for the following SPs, nothing is returned right?
* FM1028594 - SP Individual
* FM1054864 - SP LEAR DBA
* FM1061225 - SP COLIN DBA

Reply:
Can you verify your code still works for other business types?